packages/core/content-type-builder/admin/src/components/AttributeOptions/AttributeOption/index.js

```diff
@@ -43,23 +54,15 @@ const AttributeOption = ({ type }) => {
     <OptionBoxWrapper padding={4} as="button" hasRadius type="button" onClick={handleClick}>
       <Flex>
         <AttributeIcon type={type} />
-        {/* TODO: Remove after the RTE Blocks Beta release (width) */}
+        {/* TODO: Remove after the RTE Blocks Stable release (width) */}
         <Box paddingLeft={4} width="100%">
-          {/* TODO: Remove after the RTE Blocks Beta release (justifyContent) */}
+          {/* TODO: Remove after the RTE Blocks Stable release (justifyContent) */}
```

Comment:
we should remove theses comments, this code is needed for the NewBadge to work so it should remain even after the blocks stable